src/jit/compiler.h

```diff
@@ -4633,6 +4633,14 @@ class Compiler
         return fgIsUsingProfileWeights() ? fgNumProfileRuns : BB_UNITY_WEIGHT;
     }
 
+    // fgGetUnityWeight - return the "unity" block weight. This is the called count when
+    //                    edge weights have been computed (which will be BB_UNITY_WEIGHT
+    //                    in the absence of profile data), or BB_UNITY_WEIGHT otherwise.
```

Comment:
I would remove `(which will be BB_UNITY_WEIGHT in the absence of profile data),`  or reword.